docs/ui/schema-conv/sharded-migration.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Sharded migration schema
+parent: Schema Conversion Workspace
+grand_parent: SMT UI
+nav_order: 9
+---
+
+# Sharded migration schema changes
+{: .no_toc }
+
+When a sharded migration is selected by the user in the [connect to database](../connect-source.md#connect-to-database) page, SMT automatically makes some changes to the converted Spanner schema for performing sharded migrations.
+
+{: .note }
+Sharded migrations are only supported for `MySQL` currently.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Addition of `migration_shard_id`
+
+SMT adds a new column - `migration_shard_id` to each table of the source database. Each row getting copied to Spanner can be mapped back to its source. This has several advantages:
+
+1. Enables [reverse replication](../../reverse-replication/ReverseReplication.md) since each row in Spanner can be mapped back to its own source shard.
```

Comment:
We should link to the user guide instead of the setup guide.
https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/master/docs/reverse-replication/ReverseReplicationUserGuide.md